internal/manifests/manifests.go

```diff
@@ -56,58 +57,41 @@ func (m *Manifests) CreateClusterManifestInternal(ctx context.Context, params op
 	log := logutil.FromContext(ctx, m.log)
 	log.Infof("Creating manifest in cluster %s", params.ClusterID.String())
 
-	if params.CreateManifestParams.Folder == nil {
-		defaultFolder := models.CreateManifestParamsFolderManifests
-		params.CreateManifestParams.Folder = &defaultFolder
-	}
+	folder, fileName, path := m.getManifestPathsFromParameters(ctx, params.CreateManifestParams.Folder, params.CreateManifestParams.FileName)
+	log.Infof("Folder = '%s' and filename = '%s' and path = '%s'", folder, fileName, path)
 
 	// Verify that the manifests are created for a valid cluster
 	// to align kube-api and ocm behavior.
 	// In OCM, this is validated at the authorization layer. In other
 	// authorization scheme, it does not and therefore should be checked
 	// at the application level.
 	if !cluster.ClusterExists(m.db, params.ClusterID) {
-		return nil, common.NewApiError(http.StatusNotFound, fmt.Errorf("Object Not Found"))
```

Comment:
I believe the original "Object Not Found" message was purposefully created to align with the same error message returned in other cases.  It should be left as is.